modules/localization/conf/localization.conf

```diff
@@ -12,7 +12,7 @@
 # Enable lidar-based localization.
 # type: bool
 # defalult: true
---enable_lidar_localization=true
+--enable_lidar_localization=false
 
```

Comment:
Please do not change the default value.

Reply:
I have already modified the comment you left, please review again.